dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/AddVirtualProductToCartTest.php

```diff
@@ -0,0 +1,208 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\GraphQl\Quote;
+
+use Magento\Quote\Model\QuoteFactory;
+use Magento\TestFramework\Helper\Bootstrap;
+use Magento\TestFramework\TestCase\GraphQlAbstract;
+use Magento\Quote\Model\QuoteIdToMaskedQuoteIdInterface;
+use Magento\Quote\Model\ResourceModel\Quote as QuoteResource;
+use Magento\Catalog\Api\ProductCustomOptionRepositoryInterface;
+
+class AddVirtualProductToCartTest extends GraphQlAbstract
+{
+    /**
+     * @var QuoteFactory
+     */
+    private $quoteFactory;
+    /**
+     * @var QuoteResource
+     */
+    private $quoteResource;
+
+    /**
+     * @var QuoteIdToMaskedQuoteIdInterface
+     */
+    private $quoteIdToMaskedId;
+
+    /**
+     * @var ProductCustomOptionRepositoryInterface
+     */
+    private $productCustomOptionsRepository;
+
+    /**
+     * @inheritdoc
+     */
+    protected function setUp()
+    {
+        $objectManager = Bootstrap::getObjectManager();
+        $this->quoteResource = $objectManager->get(QuoteResource::class);
```

Comment:
Please replace on object `\Magento\Quote\Model\QuoteIdToMaskedQuoteIdInterface`

Reply:
Could you clarify this point please? I believe this part has been refactored already